README.rst

```diff
@@ -1,817 +1,54 @@
-=============
 openwisp-ipam
 =============
 
 .. image:: https://github.com/openwisp/openwisp-ipam/actions/workflows/ci.yml/badge.svg?branch=master
-  :target: https://github.com/openwisp/openwisp-ipam/actions/workflows/ci.yml?query=workflow%3AOpenWISP+Ipam+CI+Build%22
-  :alt: CI Build status
+    :target: https://github.com/openwisp/openwisp-ipam/actions/workflows/ci.yml?query=workflow%3AOpenWISP+Ipam+CI+Build%22
+    :alt: CI Build status
 
 .. image:: https://coveralls.io/repos/openwisp/openwisp-ipam/badge.svg
-  :target: https://coveralls.io/r/openwisp/openwisp-ipam
-  :alt: Coverage
+    :target: https://coveralls.io/r/openwisp/openwisp-ipam
+    :alt: Coverage
 
 .. image:: https://img.shields.io/pypi/v/openwisp-ipam
-  :target: https://pypi.org/project/openwisp-ipam
-  :alt: PyPI
+    :target: https://pypi.org/project/openwisp-ipam
+    :alt: PyPI
 
 .. image:: https://img.shields.io/librariesio/release/github/openwisp/openwisp-ipam
-  :target: https://libraries.io/github/openwisp/openwisp-ipam#dependencies
-  :alt: Dependency monitoring
+    :target: https://libraries.io/github/openwisp/openwisp-ipam#dependencies
+    :alt: Dependency monitoring
 
 .. image:: https://github.com/openwisp/openwisp-ipam/raw/docs/docs/subnet_demo.gif
-  :alt: Feature Highlights
```

Comment:
why remove the alt?

Reply:
It is present there, just GitHub showing changes in a weird way.